### ISSUE TITLE
Entity.mergeData not working as expected

### ISSUE THREAD

**Issue by tarun-dugar** [facebook/draft-js#753]
Hi, needed to understand how `Entity.mergeData` works. So I have the following `DraftEntityInstance` having entity key `1`:

```
    {
        'data': {
            'mention': Map({
                'text': 'Matthew Russell'
            })
        }
    }
```

Now, I need to change the text property, so I use `mergeData` like this:

```
const newEntityInstance = Entity.mergeData(1, { 'text': 'something' });
```

However, when I check `newEntityInstance`, it still has text set to 'Matthew Russell'. What am I doing wrong?

**Comment by davidchang** [comment]
sorry for late  response @tarun-dugar - this is kinda rough, there was a breaking change in which Entity operations were actually moved onto ContentState, done here: https://github.com/facebook/draft-js/pull/376/files#diff-d9ae22fa57662af1ffca8b789c559f3e But the documentation is currently a bit stale, and the code hasn't been released until version 0.10 comes out (we're currently on 0.9.1).

As for the specific text issue, do you just need to do the following?

    const newEntityInstance = Entity.mergeData(1, { 'mention': Map({ 'text': 'something' }) });

**Comment by spicyj** [comment]
Closing out old issues with no response – let us know if this should be reopened!
